recipes/tinycthreadpool/all/conanfile.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+from conan import ConanFile
+from conans import CMake, tools
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+
+required_conan_version = ">=1.53.0"
+
+
+class TinyCThreadPoolConan(ConanFile):
+    name = "tinycthreadpool"
+    description = "Portable C thread pool"
+    license = "BSD-2-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bennyhuo/tinycthreadpool"
+    topics = ("thread-pool", "threading", "pure-c")
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "CMakeDeps"
+    
+    def configure(self):
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure(source_folder="src")
+        cmake.build()
+
+    def package(self):
+        self.copy("*.h", dst="include", src="src")
```

Comment:
As you are the author of that project, you could consider add `install()` support to your cmake file, so you don't need to copy pattern by pattern 😉